gumps/Gump_manager.cc

```diff
@@ -22,23 +22,21 @@
 #  include <config.h>
 #endif
 
+#include "Configuration.h"
+#include "exult.h"
+#include "Gump.h"
+#include "Gump_manager.h"
+#include "gamewin.h"
+
 #ifdef __GNUC__
 #	pragma GCC diagnostic push
 #	pragma GCC diagnostic ignored "-Wold-style-cast"
 #endif    // __GNUC__
-#include "SDL_events.h"
-#include "SDL_keyboard.h"
 static const Uint32 EXSDL_TOUCH_MOUSEID=SDL_TOUCH_MOUSEID;
 #ifdef __GNUC__
 #	pragma GCC diagnostic pop
 #endif    // __GNUC__
```

Comment:
Shouldn't the pragmas and ifdefs be removed as well?

Reply:
I am afraid not because `SDL_TOUCH_MOUSEID` is a macro with a C style cast in it :
```
#define SDL_TOUCH_MOUSEID ((Uint32)-1)
```
The whole point of storing into `EXSDL_TOUCH_MOUSEID` is to hide this cast in a safe location in a `-Wno-old-style-cast` frame  and use `EXSDL_TOUCH_MOUSEID` everywhere else.